trapdata/antenna/tests/test_dataloader_hygiene.py

```
@@ -0,0 +1,129 @@
+``get_rest_dataloader()`` must apply the ``multiprocessing_context`` and
+``timeout`` knobs from settings. Without these, the DataLoader inherits the
+parent's heap via ``fork`` (leaking CUDA / pinned-memory state into shared
+memory) and silently hangs forever when a subprocess dies mid-batch.
+
+The RSS-growth side of the regression is already covered by
+``test_memory_leak.py``; this file focuses on the *configuration* surface so
+the fix can't be silently regressed by a future refactor of
+``get_rest_dataloader``.
+"""
+
+import multiprocessing
```

Flake8 (7.3.0)

[error] 'multiprocessing' imported but unused

(F401)